.github/workflows/mcdr.yml

```diff
@@ -0,0 +1,42 @@
+name: build MCDReforged
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * 1"
+  push:
+    branches:
+      - master
+    paths:
+      - mcdr/**
+jobs:
+  push:
+    name: "games:mcdr_java_${{ matrix.tag }}"
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        tag:
+          - '8'
+          - '11'
+          - '16'
+          - '17'
+    steps:
+      - uses: actions/checkout@v3
+      # Setup QEMU for ARM64 Build
+      - uses: docker/setup-qemu-action@v2          
+      - uses: docker/setup-buildx-action@v2
+        with:
+          buildkitd-flags: --debug
+      - uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.REGISTRY_TOKEN }}
+      - uses: docker/build-push-action@v3
+        with:
+          context: ./mcdr
+          file: ./mcdr/${{ matrix.tag }}/Dockerfile
+          platforms: linux/amd64
```

Comment:
You have the image dockerfiles set up in a way that it looks like they are multi-arch but this is amd64 only. You should probably have it set as `platforms: linux/amd64,linux/arm64`

Reply:
This would make this in like with the java workflow - https://github.com/parkervcp/yolks/blob/master/.github/workflows/java.yml